organization.go

```diff
@@ -4,10 +4,20 @@ import (
 	"fmt"
 )
 
+// OrganizationSearchResponse struct
+type OrganizationSearchResponse struct {
+	Organizations []Organization `json:"results,omitempty"`
+	NextPage      *string        `json:"next_page,omitempty"`
+	PreviousPage  *string        `json:"previous_page,omitempty"`
+	Count         *int           `json:"count,omitempty"`
+}
+
+// OrganizationWrapper struct
 type OrganizationWrapper struct {
 	Organization *Organization `   json:"organization"`
 }
 
+// OrganizationResponse struct
 type OrganizationResponse struct {
 	Organizations Organization `json:"organizations,omitempty"`
```

Comment:
That doesn't seem OK. Shouldn't that be `Organizations []Organization 'json:"organizations,omitempty"'`?
If it should we should probably settle on the same naming as for ticket collections.